mlir/test/mlir-miopen-driver/e2e/padding_kernel_gemmK.mlir

```diff
@@ -0,0 +1,19 @@
+// RUN: mlir-miopen-driver --operation conv2d -t f32 -p=false  -fil_layout=gkcyx -in_layout=ngchw -out_layout=ngkhw -batchsize=256 -groupsize=1 -in_channels=3 -out_channels=64 -in_h=224 -in_w=224 -fil_h=7 -fil_w=7 --dilation_h=1 --dilation_w=1 --padding_h=3 --padding_w=3 --conv_stride_h=2 --conv_stride_w=2 -pv -c | mlir-rocm-runner --shared-libs=%rocm_wrapper_library_dir/librocm-runtime-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext --entry-point-result=void | FileCheck %s --check-prefix=CHECK_RESNET50_CONFIG1
```

Comment:
please add the following combinations:
- `-rand 1`
- `-x2`
- `-t f16`

also please work with @yiqian1 to enable these configs in nightly CI runs.

Reply:
is default host already gfx908?

Reply:
NO. We have 2 CI nodes, one on gfx900, the other on gfx908. CHECK with @yiqian1 on how to automatically set `-x2`.